integration-tests/main/src/main/java/io/quarkus/it/opentracing/OpenTracingResource.java

```diff
@@ -14,4 +18,11 @@ public String getTest() {
         return "TEST";
     }
 
+    @GET
+    @Path("/version")
```

Comment:
`/jaegerversion` would be better since it's the Jaeger client version and not OpenTracing version